impala/dbapi.py

```diff
@@ -172,14 +181,21 @@ def connect(host='localhost', port=21050, database=None, timeout=None,
                 "'{0}' is not a supported protocol; only HiveServer2 is "
                 "supported".format(protocol))
 
+    if auth_cookie_names is not None:
```

Comment:
I think this should be:
if auth_cookie_names is not None and http_cookie_names is None:

We should consider using the value of 'auth_cookie_names' only if 'http_cookie_names' is not explicitly set.

Reply:
Fixed it.